### PR TITLE
Update for Swift 4. Refactor

### PR DESCRIPTION
**Pull request by igorkislyuk**

### CODE REVIEW

File: LeadKitAdditions/Sources/Controllers/PassCode/Model/PassCodeValidationResult.swift [hunk 1]
```diff
@@ -24,33 +24,35 @@
...
-    public var isValid: Bool {
+}
+
+public extension PassCodeValidationResult {
+    var isValid: Bool {
         switch self {
         case .valid:
             return true
-        default:
+        case .invalid(_):
```
Comment (petropavel13) [inline LeadKitAdditions/Sources/Controllers/PassCode/Model/PassCodeValidationResult.swift]: remove parameter? (` .invalid:`)

File: LeadKitAdditions/Sources/Controllers/PassCode/Model/PassCodeValidationResult.swift [hunk 2]
```diff
@@ -24,33 +24,35 @@
...
         }
     }
 
-    public var passCode: String? {
+    var passCode: String? {
         switch self {
         case let .valid(passCode):
             return passCode
-        default:
+        case .invalid(_):
```
Comment (petropavel13) [inline LeadKitAdditions/Sources/Controllers/PassCode/Model/PassCodeValidationResult.swift]: same

File: LeadKitAdditions/Sources/Controllers/PassCode/Model/PassCodeValidationResult.swift [hunk 3]
```diff
@@ -24,33 +24,35 @@
...
     }
 
-    public var error: PassCodeError? {
+    var error: PassCodeError? {
         switch self {
-        case let .inValid(error):
+        case let .invalid(error):
             return error
-        default:
+        case .valid(_):
```
Comment (petropavel13) [inline LeadKitAdditions/Sources/Controllers/PassCode/Model/PassCodeValidationResult.swift]: same

**Comment by scoreyou** [comment]
Increment version im podspec?

**Comment by igorkislyuk** [comment]
@scoreyou Done.

**Pull request merged**